ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceTlsTest.java

```diff
@@ -185,15 +186,15 @@ public void shutdownServer() {
   }
 
   @Test
-  public void netVersionSuccessfulOnTls() throws Exception {
+  public void netVersionSuccessfulOnTlsWithHttp2() throws Exception {
```

Comment:
Is it possible to have a test for http 1 as well? As all existing tests will be implicitly testing http/2 now that we have http 2 support in our service

Reply:
Added testing against both http1_1 and http2.